src/StaticFAQ/FAQExtraInfo/FAQExtraInfoButton.js

```diff
@@ -71,6 +83,7 @@ const FAQExtraInfoButton = ({ category, children, history }: Props) => {
                 <button
                   className={`extraInfoRadioButton ${isActive ? 'active' : ''}`}
                   onClick={() => {
+                    tracker[category]();
```

Comment:
IMHO `category === 'baggage' ? trackBaggage : trackBoardingPass` is more descriptive & easier for flow